### PR TITLE
feat: add symfony 7 support

### PR DESCRIPTION
**Pull request by ekadjSensio**

### CODE REVIEW

**Comment by Yondz** [comment]
Seems okay to me, should be tagged 3.0.0 once merged and we will need to test this using echonet :)

**Review by Yondz — APPROVED** [review]

File: src/LightSaml/SymfonyBridgeBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -19,18 +20,11 @@ class Configuration implements ConfigurationInterface
...
     {
         $treeBuilder = new TreeBuilder('light_saml_symfony_bridge');
-
-        if (method_exists($treeBuilder, 'getRootNode')) {
-            $root = $treeBuilder->getRootNode();
-        } else {
-            // BC layer for symfony/config 4.1 and older
-            $root = $treeBuilder->root('light_saml_symfony_bridge');
-        }
+        $root = $treeBuilder->getRootNode();
```
Comment (Jean-Beru) [inline src/LightSaml/SymfonyBridgeBundle/DependencyInjection/Configuration.php:27]: To remove the last PHPStan error :
```suggestion
        /** @var ArrayNodeDefinition $root */
        $root = $treeBuilder->getRootNode();
```
(ArrayNodeDefinition is return if the second argument of `TreeBuilder::__construct` is `array`, which is also the default value)

File: phpstan.neon [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-    level: 1
+    phpVersion: 80200
+    level: 5
     paths:
         - src
         - tests
 
     ignoreErrors:
-        - '#Instantiated class Symfony\\Component\\DependencyInjection\\DefinitionDecorator not found.#'
+        - '#Method LightSaml\\SymfonyBridgeBundle\\Bridge\\Container\\ServiceContainer::getLogoutSessionResolver\(\) has invalid return type LightSaml\\Resolver\\Logout\\LogoutSessionResolverInterface#'
```
Comment (Jean-Beru) [inline phpstan.neon]:
```php
    public function getLogoutSessionResolver(): never
    {
        throw new \LogicException('Not implemented');
    }
```
Should solve this issue.

**Review by Jean-Beru — APPROVED** [review]

File: src/LightSaml/SymfonyBridgeBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (ekadjSensio) [reply src/LightSaml/SymfonyBridgeBundle/DependencyInjection/Configuration.php:27]: I've tried this; got the [following error](https://github.com/phpstan/phpstan/issues/844) and chose the easiest solution  :smile:

Comment (Jean-Beru) [reply src/LightSaml/SymfonyBridgeBundle/DependencyInjection/Configuration.php:27]: Arf... That could be a great PR on the phpstan-symfony repo!

**Pull request merged**